### DISCUSSION TITLE
我想隐藏左上角这个模型标签，想请教一下应该怎么做

### DISCUSSION THREAD

**Discussion by Gilgamesh99** [GaiZhenbiao/ChuanhuChatGPT#754] [Q&A]
![QQ截图20230515133519](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/97493579/0add3157-9a05-4242-8b14-966e94b38a9f)

**Comment by Keldos-Li** [comment]
确实可以隐藏，我也知道怎么做，但为什么要这么干呢。

<img width="688" alt="image" src="https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/23137268/2d674ce7-cbfb-4a69-a8d6-acf015fcc54e">

**Reply by Gilgamesh99** [reply]
谢谢你的回复！虽然我明白这可能不是所有人都需要的功能，但对于我来说，我想学习一下大佬的代码，做一些小修改。如果你愿意分享一下怎么做，我将非常感激！
